### ISSUE TITLE
[Bug]: SubtleCrypto.importKey crash with "Cannot read properties of undefined" on Electron 27 Beta

### ISSUE THREAD

**Issue by threema-danilo** [electron/electron#40002]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

27.0.0-beta.7

### What operating system are you using?

Other Linux

### Operating System Version

Arch Linux: Linux jabba 6.4.12-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 24 Aug 2023 00:38:14 +0000 x86_64 GNU/Linux

### What arch are you using?

x64

### Last Known Working Electron version

26.2.2

### Expected Behavior

Web Crypto APIs should work in the worker.

### Actual Behavior

The call to `webcrypto.subtle.importKey(...)` fails in the worker on Electron 27.0.0-beta.7, but not on 26.2.2.

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'prototype')
    at isSharedArrayBuffer (node:internal/crypto/webidl:187:57)
    at Object.BufferSource (node:internal/crypto/webidl:208:9)
    at SubtleCrypto.importKey (node:internal/crypto/webcrypto:585:36)
    at worker.js:6:18
```

### Testcase Gist URL

https://gist.github.com/threema-danilo/a53a94800e258a0c2a6adb67ac7bfa72

### Additional Information

Check out the fiddle. Start it and open the debug console. On Electron 26.2.2, it prints "AES key was loaded". On Electron 27.0.0-beta.7, it prints the exception.

**Comment by Hedwig188** [comment]
Got the same error using the above test case on windows.

**Comment by threema-danilo** [comment]
Fix confirmed for 27.0.0, thanks!
